config/log-viewer.php

```diff
@@ -22,6 +22,9 @@
         'extension' => Filesystem::PATTERN_EXTENSION, // '.log'
     ],
 
+    // parse all the files in the log path, regardless of their naming
+    'parse_all_files_in_log_path' => false,
```

Comment:
Can you change the key to `parse-all-files-in-log-path` ?

Reply:
Sure I can change the key. But about the test I do not know how to do that yet.
Flipping this switch will definately break existing tests. The only thing I can think of now is to include this feature switch also in ...all... testcases that use hardcoded naming
So I need some help how to sturcture coding these tests

Reply:
You need to add new tests like:

```php
/** @test **/
public function it_can_load_all_log_files() 
{
    // Assert before switching

    $this->app['config']->set('log-viewer.parse-all-files-in-log-path', true);

    // Assert after switching
}
```

You can also create a new fixture folder that contains log files for your tests and switch it with `$this->logViewer->setPath(storage_path('my-path-logs'));`

Path: https://github.com/ARCANEDEV/LogViewer/tree/master/tests/fixtures
Example: https://github.com/ARCANEDEV/LogViewer/blob/master/tests/LogViewerTest.php#L312